Packs/Core/ReleaseNotes/2_0_9.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+Fixed an issue where the `core-list-risky-users` and `core-list-risky-hosts` commands in playbooks would fail when the Identity Threat Module was disabled or the license was missing.
```

Comment:
```suggestion
Fixed an issue where the `core-list-risky-users` and `core-list-risky-hosts` commands would fail when the Identity Threat Module was disabled or the license was missing.
```
this wasn't just in playbooks, right?

Reply:
Done